src/components/ui/DataGrid/SingleActionDataGrid.svelte

```diff
@@ -184,6 +184,9 @@
         </ContextMenu.Item>
       </div>
     {/if}
+
+    <slot name="context-menu-alt" />
```

Comment:
This is no longer needed if it is removed from `BulkActionDataGrid` as this is meant to mirror that change.